com.Cisco.PacketTracer.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Packet Tracer
+GenericName=Cisco Packet Tracer
+Comment=Cisco Packet Tracer
+Icon=com.Cisco.PacketTracer
+Exec=/app/opt/pt/packettracer
```

Comment:
the executable should be in the path (a symlink is OK) and this should be just the executable name

Reply:
Updated